inbm/dispatcher-agent/tests/unit/source/test_ubuntu_source_cmd.py

```diff
@@ -210,7 +210,7 @@ def test_add_app_with_gpg_key_successfully(self):
     def test_add_app_deb_822_format_successfully(self):
         try:
             params = ApplicationAddSourceParameters(
-                file_name="google-chrome.sources",
+                source_list_file_name="google-chrome.sources",
                 sources="X-Repolib-Name: Google Chrome"
                         "Enabled: yes"
                         "Types: deb"
```

Comment:
**Suggestion:** Ensure that concatenated strings within the 'sources' parameter are properly separated by spaces to avoid accidental string merging. [bug]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            params = ApplicationAddSourceParameters(
                source_list_file_name="google-chrome.sources",
                sources="X-Repolib-Name: Google Chrome "
                        "Enabled: yes "
                        "Types: deb "
            )
```